docs/manifest.json

```diff
@@ -816,9 +816,15 @@
 		"parent": "components"
 	},
 	{
-		"title": "DropdownMenu",
-		"slug": "dropdown-menu",
-		"markdown_source": "../packages/components/src/dropdown-menu/README.md",
+		"title": "V1",
```

Comment:
The docs manifest script doesn't seem to work well with "version" folders, as it uses only the last folder name (ie. `v1`) to create titles and slugs.

Should we rename folders to `dropdown-menu/dropdown-menu-v1` and `dropdown-menu/dropdown-menu-v2` ? Any other ideas?

Reply:
Agreed, done in https://github.com/WordPress/gutenberg/pull/49473/commits/5768fe72a6f38f5a55d793748ff8c961b5e20515